src/sentry/models/groupsubscription.py

```diff
@@ -94,6 +94,60 @@ def subscribe(self, group, user, reason=GroupSubscriptionReason.unknown):
         except IntegrityError:
             pass
 
+    def subscribe_actor(self, group, actor, reason=GroupSubscriptionReason.unknown):
+        from sentry.models import User, Team
+
+        if isinstance(actor, User):
+            return self.subscribe(group, actor, reason)
+        elif isinstance(actor, Team):
```

Comment:
A style nit here:

Since each of these conditions return, you don't need the "else" bits.

```python
if isinstance(actor, User):
    return self.subscribe(group, actor, reason)

if isinstance(actor, Team):
    ...
    return self.bulk_subscribe(...)

raise NotImplementedError('Unknown actor type: %r' % type(actor))
```

The value to me here is reduced indentation levels which makes itself a bit more readable.

Reply:
good call